packages/upstream/rpclib-lwt.5.8.0/url

```diff
@@ -1,2 +1,2 @@
 archive: "https://github.com/mirage/ocaml-rpc/archive/v5.1.0.tar.gz"
-checksum: "97f526cf3c8c8fca033df1175d4d23df"
+checksum: "4dc2050aeba98fe3a538377b17205348"
```

Comment:
Same question as above.

Reply:
Well-spotted, I forgot to update those urls.